src/components/molecules/SearchField/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { Input, InputGroup, InputLeftElement } from '@chakra-ui/react'
+import { Search2Icon } from '@chakra-ui/icons'
+import { useSearchParams } from 'next/navigation'
+import { useCallback, useRef, useState } from 'react'
+
+export default function SearchField() {
```

Comment:
Copied codes from the folowing:
https://github.com/microcmsio/nextjs-simple-blog-template/blob/main/components/SearchField/index.tsx